README.md

```diff
@@ -74,6 +116,14 @@ compile 'ch.aaap:harvest-client:$version'
     ```   
 * Run ```gradle build``` in the root directory of the repository
 
+### Creating a test Harvest account
+go through creating a new account.
+My Profile -> link under work email to your Harvest ID settings -> Developers -> create new personal Token (name does not matter)
```

Comment:
link should work, right? https://id.getharvest.com/developers

Reply:
yes that's the current link